docs/source/developers/benchmarks.rst

```diff
@@ -59,7 +59,7 @@ Sometimes, it is required to pass custom CMake flags, e.g.
 .. code-block:: shell
 
   export CC=clang-8 CXX=clang++8
-  archery benchmark run --cmake-extras="-DARROW_USE_SIMD=ON"
+  archery benchmark run --cmake-extras="-DARROW_SIMD_LEVEL=NONE"
```

Comment:
I guess it should be SSE4_2 as it's the default option. ARROW_USE_SIMD is on as default also. Or just rm the cmake-extras?

Reply:
[The context](https://arrow.apache.org/docs/developers/benchmarks.html#running-the-benchmark-suite) is to show how to pass extra cmake flags. Default value looks not very useful. Maybe change to AVX2?

Reply:
Get it, then ARROW_SIMD_LEVEL=NONE is fine:)